### PR TITLE
Add jest unit tests for lambda@edge

### PR DESCRIPTION
**Pull request by karthikvadla**
This PR adds jest unit tests for lambda@edge.

### CODE REVIEW

File: src/lambda-edge/shared/logger.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+/* istanbul ignore file */
+
+export enum LogLevel {
+    'none' = 0,
+    'info' = 10,
```
Comment (ottokruse) [inline src/lambda-edge/shared/logger.ts:7]: Switch info and error (was a bug I fixed on master).
Should be:
info: 30
error: 10

File: src/lambda-edge/shared/shared.ts [hunk 2]
```diff
@@ -18,18 +20,18 @@ export interface CookieSettings {
 
 const defaultCookieSettings: { [key: string]: CookieSettings } = {
     spaMode: {
-        idToken: "Path=/; Secure; SameSite=Lax",
-        accessToken: "Path=/; Secure; SameSite=Lax",
-        refreshToken: "Path=/; Secure; SameSite=Lax",
-        nonce: "Path=/; Secure; HttpOnly; SameSite=Lax"
+        idToken: 'Path=/; Secure; SameSite=Lax',
+        accessToken: 'Path=/; Secure; SameSite=L:qax',
```
Comment (ottokruse) [inline src/lambda-edge/shared/shared.ts:24]: L:qax :) typo

File: src/lambda-edge/shared/logger.ts (hunk 1 above)
Comment (karthikvadla) [reply src/lambda-edge/shared/logger.ts:7]: Sure will update.

File: src/lambda-edge/shared/shared.ts (hunk 2 above)
Comment (karthikvadla) [reply src/lambda-edge/shared/shared.ts:24]: nice catch.

**Comment by ottokruse** [comment]
This is super high on my wish list to further look at and get in. Mea culpa for the delay
